From 832ab0d5618f189ae089453d65479f78abf23f5c Mon Sep 17 00:00:00 2001
From: xiaobai <fengz181x@126.com>
Date: Sat, 24 Jun 2017 09:36:08 +0800
Subject: [PATCH] fix build error

---
 hal/audio_extn/audio_extn.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hal/audio_extn/audio_extn.h b/hal/audio_extn/audio_extn.h
index 24e0fb8..147fd69 100644
--- a/hal/audio_extn/audio_extn.h
+++ b/hal/audio_extn/audio_extn.h
@@ -535,7 +535,7 @@ int audio_extn_pm_vote (void);
 void audio_extn_pm_unvote(void);
 #endif
 
-void audio_extn_init(struct audio_device *adev);
+void audio_extn_init(struct audio_device *adev __unused);
 void audio_extn_utils_update_streams_cfg_lists(void *platform,
                                   struct mixer *mixer,
                                   struct listnode *streams_output_cfg_list,

